src/js/components/App.jsx

```diff
@@ -58,6 +70,22 @@ class App extends Component {
     const banner = bannerText
       ? <div style={style.banner}>{bannerText}</div>
       : null
+    const footerLeft = footerLeftText
+      ? <div
+        dangerouslySetInnerHTML={{__html: footerLeftText}}
```

Comment:
So, dangerouslySetInnerHTML... We want to be able to have line breaks in there. And email addresses. Other than making it overly complicated, I felt that this would be ok here.